.prettierrc

```diff
@@ -0,0 +1,8 @@
+{
+	"useTabs": true,
+	"singleQuote": true,
+	"trailingComma": "none",
+	"printWidth": 100,
+	"plugins": ["prettier-plugin-svelte"],
+	"overrides": [{ "files": "*.svelte", "options": { "parser": "svelte" } }]
```

Comment:
Whats this?

Reply:
Code formatter. Same as revanced-website for consistency

Reply:
What is "overrides"?

Reply:
Overrides let you have different configuration for certain file extensions. It will override svelte's default formatting